fastai/text/transform.py

```diff
@@ -140,6 +140,11 @@ def __setstate__(self, state:dict):
         self.itos = state['itos']
         self.stoi = collections.defaultdict(int,{v:k for k,v in enumerate(self.itos)})
 
+    def save(self, path):
+        "Save the itos in path"
+        with open(path, 'wb') as f:
```

Comment:
This can all go on the same line with `pickle.dump(self.itos, open(path,'wb'))`.